core/templates/pages/contributor-dashboard-admin-page/translation-role-editor-modal/cd-admin-translation-role-editor-modal.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="modal-header e2e-test-language-selector-modal">
+  <h4 class="modal-title">Translation reviewer roles for user {{username}}</h4>
+</div>
+<div class="modal-body">
+  <div class="modal-body">
+    <p *ngIf="assignedLanguageIds.length">User has review rights in the following languages:</p>
+    <p *ngIf="!assignedLanguageIds.length">Please assign user to atleast one language:</p>
```

Comment:
```suggestion
    <p *ngIf="!assignedLanguageIds.length">Please assign user to at least one language:</p>
```

Reply:
Though should this even be the message here? Seems valid for the editor to want to remove all languages.

Reply:
Updated the message, can you please take a look again and see if it is reasonable now?